src/plugins/intel_cpu/src/nodes/transpose.cpp

```diff
@@ -14,6 +14,32 @@ using namespace mkldnn;
 using namespace ov::intel_cpu;
 using namespace InferenceEngine;
 
+namespace {
```

Comment:
[random spot] please add tests (static and dynamic) for this test case in separate PR

Reply:
OK